analytics-example-widget/content/AnalyticsConfiguration.css

```diff
@@ -0,0 +1,31 @@
+.bowtie.widget-config-component div {
+    padding-bottom: 10px;
+}
+
+.react-container{
+    height:700px;
+}
+
+
+.widget-config-component select {
```

Comment:
i noticed the style files are css instead of scss.. intentional choice?

Reply:
Intentional - Samples steer clear of build/deploy complexity, where those aspects are not the feature. Likewise, this is why bundling and minification of script is not exercised.